server/core/src/irodsReServer.cpp

```diff
@@ -541,7 +569,6 @@ int main(int argc, char **argv)
     signal(SIGHUP, signal_exit_handler);
     signal(SIGTERM, signal_exit_handler);
     signal(SIGUSR1, signal_exit_handler);
-
     const auto sleep_time = [] {
```

Comment:
lets restore the line above L572 since the code above it is not related to the code that follows it